yt_dlp/extractor/rtve.py

```diff
@@ -282,46 +293,46 @@ class RTVELiveIE(RTVEALaCartaIE):  # XXX: Do not subclass from concrete IE
         'info_dict': {
             'id': 'la-1',
             'ext': 'mp4',
-            'title': 're:^La 1 [0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}$',
+            'title': str,
+            'live_status': 'is_live',
         },
         'params': {
             'skip_download': 'live stream',
         },
     }]
 
     def _real_extract(self, url):
-        mobj = self._match_valid_url(url)
-        video_id = mobj.group('id')
-
+        video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
-        title = remove_end(self._og_search_title(webpage), ' en directo en RTVE.es')
-        title = remove_start(title, 'Estoy viendo ')
 
-        vidplayer_id = self._search_regex(
-            (r'playerId=player([0-9]+)',
-             r'class=["\'].*?\blive_mod\b.*?["\'][^>]+data-assetid=["\'](\d+)',
-             r'data-id=["\'](\d+)'),
-            webpage, 'internal video ID')
+        data_setup = self._search_json(
+            r'<div[^>]+class="[^"]*videoPlayer[^"]*"[^>]*data-setup=\'',
```

Comment:
they single quote the json and don't escape html special chars (`"`=>`&quot;`)?

Reply:
Yes
```html
      <div class="ima f16x9 videoPlayer" 
      data-setup='{
        "id":1,
        "idAsset":"1688877",
        "isLive": "true",
        "segment": false,
        "imagen": "https://img2.rtve.es/resources/thumbslive/H_1688877.jpg",
        "hasDRM": false,
        "idLicencia": null,
        "requireLogged": true
      }'>
```